app_dart/lib/src/foundation/utils.dart

```diff
@@ -95,7 +97,10 @@ Future<List<Map<String, dynamic>>> getRepoBuilders(HttpClientProvider branchHttp
   Map<String, dynamic> builderMap;
   builderMap = json.decode(builderContent) as Map<String, dynamic>;
   final List<dynamic> builderList = builderMap['builders'] as List<dynamic>;
-  return builderList.map((dynamic builder) => builder as Map<String, dynamic>).toList();
+  return builderList
+      .map((dynamic builder) => builder as Map<String, dynamic>)
+      .where((Map<String, dynamic> element) => (element['enabled'] as bool) ?? true)
+      .toList();
```

Comment:
Do we need to add the property to the model class?

Reply:
The model class is for prod builders usage only. We don't need `enabled` flag for prod builders, right?

Reply:
> The model class is for prod builders usage only. We don't need `enabled` flag for prod builders, right?

Good point I though we were using the model class for try too.